File: LeetCode/Easy/2269. Find the K-Beauty of a Number/2026-04-27_Accepted.py
class Solution:
  def divisorSubstrings(self, num: int, k: int) -> int:
    ans = 0
    num_str = str(num)
    for i in range(0, len(num_str) - k + 1):
      divisor = int(num_str[i : i + k])
      if divisor != 0 and num % divisor == 0:
        ans += 1
    return ans

File: LeetCode/Easy/2239. Find Closest Number to Zero/2026-04-27_Accepted.cpp
class Solution {
public:
  int findClosestNumber(vector<int>& nums) {
    int min_abs_val = 100001;
    int ans;
    for (auto num : nums) {
      if (abs(num) < min_abs_val || (abs(num) == min_abs_val && num > ans)) {
        min_abs_val = abs(num);
        ans = num;
      }
    }
    return ans;
  }
};
